### DISCUSSION TITLE
[1.17]A suggestion about Lightning Rod.

### DISCUSSION THREAD

**Discussion by Sandendaty** [TechReborn/TechReborn#2539] [Ideas]
Hello,author!
Due to the update of vanilla minecraft 1.17,the lightning rod making of 3 copper ingots was added to vanilla minecraft.On one occasion,I happened to find that there was a block in your mod,which also named lightning rod.Out of curiosity,I searched the usage of this block.Then I found that there weren't any connections between the lightning rod in your mod and the one in vanilla,feeling a bit disappoint.
For the above reasons,I think it would be better if you can make compatibility of them.For example,replace the iron fence with the copper rod and put the vanilla lightning rod on the top of rod.

**Comment by Ayutac** [comment]
I am definitely in favor of using the vanilla lightning rod in the recipe of the TR lightning rod :)
